circle.yml

```diff
@@ -61,12 +61,14 @@ test:
 
         # Lint PHP
         - |
-            COMMIT_RANGE=$(echo "${CIRCLE_COMPARE_URL}" | cut -d/ -f7 | sed -e "s/\.\.\./ /")
+            COMMIT_RANGE=$(echo "${CIRCLE_COMPARE_URL}" | cut -d/ -f7)
             # Fix single commit
             if [[ $COMMIT_RANGE != *"..."* ]]; then
-              COMMIT_RANGE="${COMMIT_RANGE} ${COMMIT_RANGE}"
+              COMMIT_RANGE="$(git rev-parse --short=12 origin/develop)...${COMMIT_RANGE}"
```

Comment:
I don't really get why `origin/develop` is hardcoded ?

Reply:
`${COMMIT_RANGE}~` used instead, thanks @B-Galati ;)